core/controllers/topics_and_skills_dashboard.py

```diff
@@ -162,6 +162,33 @@ def get(self):
         self.render_json(self.values)
 
 
+class SkillsDashboardPageDataHandler(base.BaseHandler):
+    """Provides data for the user's skills dashboard page."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_topics_and_skills_dashboard
+    def post(self):
+        """Handles POST requests."""
+
+        classroom_name = self.payload.get('classroomName')
```

Comment:
Do we normally send data to the backend in lowerCamelCase? Please follow existing conventions.

Reply:
Fixed. Done.